### ISSUE TITLE
Monk->Bard Class swap

### ISSUE THREAD

**Issue by Embrelle** [svof/svof#779]
Swapping from Monk to Bard causes "[string \"Script: Setup\"]:431: attempt to index field 'tekura' (a nil value)" to appear on every line. A makeshift fix that's worked for me has been to change line 135 in the setup script from:
svo.sk               = svo.sk or {}
to
svo.sk               =  {}
which has not caused any further issues thus far.

**Comment by tsartin** [comment]
@Embrelle Are you still having this issue? And if so, does it do this on a new profile?

**Comment by Embrelle** [comment]
@tsartin Yes, just tested with a fresh profile and the problem is still there. As a note for replication I've tried this with both Shikudo and Tekura monk, and the error message disappears after restarting mudlet but will come back if I switch to monk and then to bard.

**Comment by tsartin** [comment]
@Embrelle could you post a screen shot of what it looks like right after you change your class to bard? I'm trying to figure out where the switch is failing. 

It should have something like 
"(svof): Welcome back!
(svof): Loaded & ready to go, version 52 (homepage)
...etc"

**Comment by Embrelle** [comment]
![image](https://user-images.githubusercontent.com/114719949/200421730-53fb05b7-0f1e-416b-b645-c79870e01360.png)
@tsartin

**Comment by Embrelle** [comment]
@tsartin your proposed change, to add svo.signals.gmcpcharvitals = nil to svo.classchange resolved the problem!
